core/templates/pages/creator-dashboard-page/creator-dashboard-page.component.html

```diff
@@ -64,7 +64,7 @@ <h1 class="stat-value-without-rating" [hidden]="dashboardStats.numRatings || rel
       </div>
     </mat-card>
   </div>
-  <div class="row" *ngIf="enableQuestionSuggestions">
+  <div class="row" *ngIf="undefined">
```

Comment:
What's the point of putting undefined here and below? You might as well remove the ngIf.